.github/vale/Docker/Capitalization.yml

```
@@ -0,0 +1,10 @@
+extends: existence
+message: "Please capitalize Docker."
+level: error
+ignorecase: false
+action:
+  name: replace
+  params:
+    - Docker
+tokens:
+  - 'docker(?!/cli)'
```

I guess we may need `buildx`, `compose`, `<other repos>` as well; should those be added as separate items in this list, or do we want to try capturing them in a single RegEx?

---

ya you're right, we should. Maybe this?

```suggestion
  - 'docker(?!/[a-z-]+)'
```

---

Yup; something like that seems good (but make sure to test indeed) 🙈 

I updated with your suggestion